Schedule-maker/schedule-parser.js

```diff
@@ -47,21 +47,27 @@ function parseSchedule(){
         classI.nbr = course[1].split(" ")[1].slice(1, -1);
         item.course = classI;
         var sched = tbodymain.children[i].children[1+offset].firstElementChild.innerText.split('\n');
-        fullSched = {};
-        days = sched[0].split(" ")[0].split(/(?=[A-Z])/);
-        fullSched.days = days.map(function(e){return e.toUpperCase()});
-        time = {}
-        try {
-            time.from = convertTime(sched[0].split(" ")[1]);
-            time.to = convertTime(sched[0].split(" ")[3]);
-        }
-        catch(err) {
-            // this means time is not provided
-            continue;
+        subscheds = [];
+        locs = [];
+        for (var j=0; j<sched.length/2; j++){
```

Comment:
Not sure I'f I'm completely understanding, but shouldn't this be `j += 2` instead of `j++`?

Reply:
OH F*CK you are rightttt